ssh-keyman.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License Apache 2.0 (https://www.apache.org/licenses/LICENSE-2.0.html)
+
+from getpass import getpass
+
+try:
+    import ldap
+except ImportError:
+    print 'To use this script, you need to install `python-ldap`'
+    exit(0)
+
+
+## BEGIN CONFIG
+LDAP_SERVER = 'ldap://127.0.0.1'
+BASE_DN = 'OU=Example,DC=corp,DC=example,DC=com'
+SSH_KEY_ATTR = 'sshPublicKeys'
+## END CONFIG
+
+
+try:
+    ldap.set_option(ldap.OPT_REFERRALS, 0)
+    ldap.set_option(ldap.OPT_PROTOCOL_VERSION, 3)
+    ldap.set_option(ldap.OPT_X_TLS_REQUIRE_CERT, ldap.OPT_X_TLS_ALLOW)
+    ldap.set_option(ldap.OPT_X_TLS, ldap.OPT_X_TLS_DEMAND)
+    ldap.set_option(ldap.OPT_X_TLS_DEMAND, True)
+    ldap_client = ldap.initialize(LDAP_SERVER)
+    username = raw_input('Username: ')
+    password = getpass('Password: ')
+    ldap_client.start_tls_s()
+    ldap_client.simple_bind_s(username, password)
+except ldap.LDAPError, e:
+   print e
```

Comment:
Wouldn't it be better to use logging here? Or at least something Python 3 compat?